tests/v2/test_rpc_api.nim

```diff
@@ -0,0 +1,91 @@
+import
+  std/[unittest, options, sets, tables, os, strutils],
+  stew/shims/net as stewNet,
+  json_rpc/[rpcserver, rpcclient],
+  libp2p/standard_setup,
+  libp2p/switch,
+  libp2p/protobuf/minprotobuf,
+  libp2p/stream/[bufferstream, connection],
+  libp2p/crypto/crypto,
+  libp2p/protocols/pubsub/rpc/message,
+  ../../waku/v2/waku_types,
+  ../../waku/v2/node/wakunode2,
+  ../../waku/v2/node/rpc_api/[waku_api_rpc, waku_api_types],
+  ../../waku/v2/protocol/[waku_store, message_notifier],
+  ../test_helpers
+
+template sourceDir*: string = currentSourcePath.rsplit(DirSep, 1)[0]
+const sigPath = sourceDir / ParDir / ParDir / "waku" / "v2" / "node" / "rpc_api" / "waku_api_callsigs.nim"
+createRpcSigs(RpcHttpClient, sigPath)
+
+suite "Waku v2 JSON-RPC API":
+
+  asyncTest "get_waku_v2_store_query":
+    # WakuNode setup
+    let
+      rng = crypto.newRng()
+      privkey = crypto.PrivateKey.random(Secp256k1, rng[]).tryGet()
+      bindIp = ValidIpAddress.init("0.0.0.0")
+      extIp = ValidIpAddress.init("127.0.0.1")
+      port = Port(9000)
+      node = WakuNode.init(privkey, bindIp, port, some(extIp), some(port))
+
+    waitFor node.start()
+
+    waitFor node.mountRelay(@["waku"])
```

Comment:
I think this should be the default waku topic, mostly for consistency

Reply:
Will do!

Reply:
Addressed in 4047bed